run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -ux
+status=0
+
+# global exports
+export PGPORT=55435
+
+# build extension
+make install
+
+# initialize database
+initdb -D $PGDATA
+
+# change PG's config
+echo "port = $PGPORT" >> $PGDATA/postgresql.conf
+
+# start cluster
+pg_ctl start -l /tmp/postgres.log -w || status=$?
+
+# something's wrong, exit now!
+if [ $status -ne 0 ]; then cat /tmp/postgres.log; exit 1; fi
```

Comment:
Why a separate status variable instead of just checking $? on the line 22.

Reply:
Also, please, use one-liners with &&  and { instead of `if` `then` `fi` blocks, i.e.
[[ $? -ne 0 ]] && cat /tmp/postgres.log && exit 1;

Reply:
thanks